docs/backup/08-01-velero-turorial.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Taking backup using Velero
+type: Tutorials
+---
+
+Follow this tutorial to install Velero and take a backup from Kyma cluster.
```

Comment:
We need to add a short intro with disclaimer, something like:
For scenarios of partial restore of individual applications running on Kyma, you might want to use Velero. It will create a backup of your kubernetes resources and volumes so that you can restore them on a different cluster.
Be aware that a full restore of a Kyma cluster is not supported. You need to always start from an existing Kyma installation and restore explicit resources individual.

Reply:
Added